flight_computer/packets.py

```diff
@@ -1,66 +1,65 @@
 import json
-import struct
+
+import bitstring
+from bitstring import BitArray
+
+SCHEMAS_PATH = "./packet_schemas/schemas"
+
+GENERIC_PACKET_TYPE_NUM = 1
+TELEMETRY_PACKET_TYPE_NUM = 2
+
+MAGIC_NUMBER = 99  # must be between -128 to 127
 
 
-# Represents one set of data to be sent or received
 class Packet:
     """
     A class to represent data to be sent or received with the radio.
     It maintains a dictionary mapping fields to values, which can be
-    serialized with the CircuitPython struct library.
+    serialized with the python bitstring library.
```

Comment:
Have you double-checked that this exists in CircuitPython? Although the syntax of CircuitPython is largely the same as in Python, it has much fewer libraries.